open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/ResourceMessageHandler.java

```diff
@@ -70,11 +88,18 @@ public DefaultSuccessMAP handleValidated(ResourceMAP messageAndPayload) throws R
                                     //Check if resource exists yet
                                     if (resourceStatusHandler.resourceExists(((ResourceUpdateMessage) messageAndPayload.getMessage()).getAffectedResource())
                                             || resourceStatusHandler.resourceExists(ResourcePersistenceAndIndexing.tryGetRewrittenResourceUri(messageAndPayload.getMessage().getIssuerConnector(), ((ResourceUpdateMessage) messageAndPayload.getMessage()).getAffectedResource()))) {
-                                        throw new RejectMessageException(RejectionReason.TOO_MANY_RESULTS, new Exception("The resource you are trying to post already exists. To update it, use PUT instead."));
+                                    	logger.error("The resource you are trying to post already exists. To update it, use PUT instead.");
+                                    	throw new RejectMessageException(RejectionReason.TOO_MANY_RESULTS, new Exception("The resource you are trying to post already exists. To update it, use PUT instead."));
                                     }
                                 }
                                 catch (RejectMessageException ignored)
                                 {
+                                	logger.error("Message could not be processed.");
```

Comment:
This block of error messages is incorrect. This will always occur when a new resource is registered and should not be logged as an error. Please note that the exception name "ignored" indicates that this exception does not matter and can be safely ignored.